coalib/output/ConsoleInteraction.py

```diff
@@ -321,7 +321,7 @@ def print_affected_files(console_printer,
     :param console_printer: Object to print messages on the console.
```

Comment:
@Adrianzatreanu i tried it earlier, but the message was too long. Couldn't come up with a shorter message with ConsoleInteraction.